test-em-all.sh

```
@@ -229,29 +378,27 @@ function verifyAPIs() {
     recreateComposite "$CUSTOMER_NAME" "$body" "order-service/api/orders" "POST" || return 1
 
     local ORDER_ID=$(echo ${COMPOSITE_RESPONSE} | jq .orderId)
-    
-    # Check if order ID was retrieved successfully
+      # Check if order ID was retrieved successfully
     if [[ -z "$ORDER_ID" || "$ORDER_ID" == "null" ]]; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)